zkevm-circuits/src/evm_circuit/execution/end_block.rs

```diff
@@ -143,13 +141,7 @@ impl<F: Field> ExecutionGadget<F> for EndBlockGadget<F> {
             .assign(region, offset, Value::known(total_txs))?;
         self.total_txs_is_max_txs
             .assign(region, offset, total_txs, max_txs)?;
-        let max_txs_assigned = self.max_txs.assign(region, offset, Value::known(max_txs))?;
```

Comment:
Here are some left over. Those logic already being done in line 131
`self.rw_table_padding_gadget`.